pipelines/fit_pyrenew_model.py

```diff
@@ -30,10 +33,21 @@ def fit_and_save_model(
             "to seed :func:`jax.random.key`"
         )
 
-    (model_data, priors) = get_model_data_and_priors_from_dir(model_run_dir)
-    (my_model, my_data) = build_pyrenew_hew_model(
-        model_data,
+    priors = get_priors_from_dir(model_run_dir)
+    my_data = PyrenewHEWData.from_json(
+        json_file_path=Path(model_run_dir)
+        / "data"
+        / "data_for_model_fit.json",
+        fit_ed_visits=fit_ed_visits,
+        fit_hospital_admissions=fit_hospital_admissions,
+        fit_wastewater=fit_wastewater,
+    )
+    model_params = PyrenewHEWParam.from_json(
+        Path(model_run_dir) / "model_params.json"
+    )
+    my_model = build_pyrenew_hew_model(
```

Comment:
I think it would be useful to remove the `get_priors_from_dir` function and bundle all of this in a `build_pyrenew_hew_model_from_dir` function, which has the relative paths as arguments and good defaults.